app/slave/subjob_executor.py

```diff
@@ -2,7 +2,8 @@
 import shutil
 import time
 
-from app.master.subjob import Subjob
+from app.master.build import BuildArtifact
```

Comment:
Yup, was thinking about whether to do that in this PR or another one. But might as well do it in this one.